pkg/config/config_test.go

```diff
@@ -149,7 +149,7 @@ func TestParseConfiguration(t *testing.T) {
 	if cfg.GetEbpfEngineLoaderPath() == "" {
 		t.Errorf("expected empty ebpf engine loader path but got %v", cfg.GetEbpfEngineLoaderPath())
 	}
-	if cfg.GetUpdateDataPeriod() != 30*time.Second {
+	if cfg.GetUpdateDataPeriod() != 60*time.Second {
```

Comment:
```suggestion
	if cfg.GetUpdateDataPeriod() != 1*time.Minute {
```
This should be the prepare way of testing